packages/client/__tests__/fixtures/peers.js

```diff
@@ -0,0 +1,47 @@
+const peer1 = {
+  ip: '1.1.1.1',
+  port: 4002,
+  version: '1.1.1',
+  errors: 0,
+  os: 'linux',
+  height: 3663605,
+  status: 'OK',
+  delay: 217
+}
+const peer2 = {
+  ip: '2.2.2.2',
+  port: 4002,
+  version: '1.1.1',
+  errors: 0,
+  os: 'linux',
+  height: 3663608,
+  status: 'OK',
+  delay: 21
+}
+const peer3 = {
+  ip: '3.3.3.3',
+  port: 4002,
+  version: '1.1.1',
+  errors: 0,
+  os: 'linux',
+  height: 3663600,
+  status: 'OK',
+  delay: 19
+}
+const peer4 = {
+  ip: '4.4.4.4',
+  port: 4002,
+  version: '1.1.1',
+  errors: 0,
+  os: 'linux',
+  height: 3663605,
+  status: 'OK',
+  delay: 17
+}
+const peers = [peer1, peer2, peer3, peer4]
```

Comment:
Might it be worth just moving this to `exports.peers = [peer1, peer2, peer3, peer4]`? Just saves storing it when it's only used once